samples/bluetooth/broadcast_audio_assistant/README.rst

```diff
@@ -0,0 +1,51 @@
+.. zephyr:code-sample:: bluetooth_broadcast_audio_assistant
+   :name: Bluetooth: Broadcast Audio Assistant
+   :relevant-api: bt_bap
+
+   Bluetooth: Broadcast Audio Assistant
+
+Overview
+********
+
+Application demonstrating the LE Audio broadcast assistant functionality.
+
+The sample will automatically try to connect to a device in the BAP Scan Delegator
+role (advertising support for the Broadcast Audio Scan Service (BASS)).
+It will then search for a broadcast source and (if found) add the broadcast ID to
+the BAP Scan Delegator.
+
+This sample can be found under
+:zephyr_file:`samples/bluetooth/broadcast_audio_assistant` in the Zephyr tree.
+
+Check the :ref:`bluetooth samples section <bluetooth-samples>` for general information.
+
+Requirements
+************
+
+* BlueZ running on the host, or
+* A board with Bluetooth Low Energy 5.2 support
```

Comment:
Isn't there more to list here? Like "another board running XYZ", "a device capable of blah such as earbuds"? This sample is not meant to be tested alone, right?

Reply:
I can list the source and sink, true ;)